### PR TITLE
Feature/a rice blogs/guns vs butter.html

### PR DESCRIPTION
**Pull request by gooddogthedog**
added homepage link
used more descriptive branch name
embedded a rice's sources in text superscripts
formatted html w/ online html formatter
added free clipart image for blog image

### CODE REVIEW

**Comment by Azurras** [comment]
The image on the homepage is too big. It should be 253px x 193px. Yours is 253px x 253px.

**Comment by Azurras** [comment]
There are two titles on this blog. One as the regular title and one in the lead paragraph.

**Comment by Azurras** [comment]
Also, the document is still not properly formatted.

**Comment by gooddogthedog** [comment]
> 
> 
> Also, the document is still not properly formatted.

if you will show me an example of how you want it formatted, i will follow it. i'm probably not going to download visual studio to format it

**Comment by gooddogthedog** [comment]
> 
> 
> There are two titles on this blog. One as the regular title and one in the lead paragraph.

A Rice's blog had a subtitle - seemed fine for the lead

**Comment by Azurras** [comment]
> > Also, the document is still not properly formatted.
> 
> if you will show me an example of how you want it formatted, i will follow it. i'm probably not going to download visual studio to format it

Look at the previous blog.

**Comment by Azurras** [comment]
> > There are two titles on this blog. One as the regular title and one in the lead paragraph.
> 
> A Rice's blog had a subtitle - seemed fine for the lead

lead paragraph is for lead paragraph. If we want a subtitle then we should find something in bootstrap to support it.

**Comment by Azurras** [comment]
I'm okay with everything here, but the formatting. Look at the other blog, notice that it has a character limit for each line. It doesn't require lots of side scrolling in order to read.

**Comment by Azurras** [comment]
This document is formatted for 3 spaces. No where in the industry is 3 spaces used. 4 spaces and 4 character tabs are most commonly used for Java code. Like every other file in this repo, use 4 space indentation.

File: src/main/resources/static/blog/arice/guns_vs_butter.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+               
+                <p>Many in the U.S. have noticed the plethora of 
+                    social programs and new infrastructure European nations 
+                    enjoy. These benefits include universal healthcare, a 
+                    robust 
+                    education system, housing, renewable energy infrastructure, 
+                    etc.  Many notice these benefits and ask, “Why don’t we 
+                    have that?” The reason is that we are providing the 
+                    <i>guns</i> for them while they buy themselves <i>butter</
+                    i>. It is that simple: the butter our allies relish in is
```
Comment (Azurras) [inline src/main/resources/static/blog/arice/guns_vs_butter.html:120]: On line 120, notice how it <I>butter</ and then it's a new line? The browser won't recognize that tags ending with that new line there.

**Comment by gooddogthedog** [comment]
yeehaw.

**Pull request merged**